src/odinapi/templates/index.html

```diff
@@ -60,6 +60,24 @@ <h2 class="featurette-heading">Version 3 released<span class="text-muted"> 2020-
               products.
             </li>
           </ul>
+          DOIs:
```

Comment:
Om du ska ha den här rubriken tycker jag att doiorna borde stå i klartext också, inte bara länka. Alternativt byt rubrik till något i stil med
```suggestion
          DOI links:
```